pt/lessons/basics/enum.md

```diff
@@ -207,3 +207,62 @@ iex> Enum.uniq([1, 2, 3, 2, 1, 1, 1, 1, 1])
 iex> Enum.uniq_by([%{x: 1, y: 1}, %{x: 2, y: 1}, %{x: 3, y: 3}], fn coord -> coord.y end)
```

Comment:
Just a small change: the version in this markdown header needs to be updated to `1.6.1`.

Reply:
Done! Thank you.